cc/modules/protocol/mpc/snn/src/impl/division.cpp

```diff
@@ -288,7 +288,59 @@ int ReciprocalDiv::ReciprocalDivfor2(
 
     vector<mpc_t> quotient_vec = shared_quotient_vec;
 
-    
+//we should limit the denominator to expand the scope of computation and the accuracy.
+    	vector<mpc_t> SHARED_ONE(vec_size, 0);
+	 if(partyNum == PARTY_A) {
+	SHARED_ONE = vector<mpc_t>(vec_size, FloatToMpcType(1));
```

Comment:
these block code in `if` should be indented.

Reply:
OK ,I have rectified it.